src/helpers/mailer.js

```diff
@@ -4,16 +4,21 @@ import config from '../config/config';
 
 module.exports = {
 
-    sendEmail(res, email, text, token, next) {
+    sendEmail(res, email, subject, text, attributes, next) {
         const options = {
             from: util.format('"%s"', config.mailer.fromAddress),
             to: email,
-            subject: 'Password Reset',
+            subject,
             text,
+            // attributes is needed here only for the purpose of testing, use this to pass in
+            // tokens or other values needed for validation from the response.
+            attributes,
```

Comment:
Is this argument indeed only for testing? If so, is there a different way testing can be accomplished?

Reply:
Not without an extensive re-write